#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Adding empty arm framestate.c not to build for dwarf2.
# DP: Author: Philip Blundell <pb@nexus.co.uk>
# DP: Upstream status: Debian-Specific
# DP: Status Details: ARM uses sjlj exceptions, not dwarf2 unwind.
# DP:		This patch adds empty framestate.c for arm to avoid
# DP:		dwarf2 build failure.
# DP: Date: 2003-07-19

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
--- /dev/null   2003-01-13 18:16:28.000000000 +0900
+++ sysdeps/arm/framestate.c    2003-07-16 23:21:34.000000000 +0900
@@ -0,0 +1,2 @@
+
+
